README.md

```diff
@@ -1,139 +1,73 @@
-
-mmj2's capabilities are invoked using "RunParm" commands that form a language, albeit without looping or "if" statements. Thus, it is easy to enhance and extend mmj2, even for "one offs": code a new program, a new "RunParm" command or two, and modify the mmj.util.BatchMMJ2.java program, and one can immediately gain programmatic access to a fully parsed, proof-verified Metamath database loaded into memory and ready for use. (Note: the [MMJ2 Service][11] Feature allows your program to call mmj2 or to be called by mmj2.)
-
-5. Shortcomings of mmj2:
-
-
- * It is not a "prover".
- * It does not update Metamath .mm databases.
- * Its grammatical analysis module does minimal checking for grammatical ambiguity (this section is stubbed out -- very difficult, requiring lots of work in special cases because the general case is impossible to prove :).
- * It is a single-user system, not designed for concurrent access throughout (but you can run more than one instance of mmj2's Proof Assistant GUI simultaneously.)
+# README for mmj2
+
+mmj2 is a proof assistant for the Metamath language.
+Metamath is a language that lets you express mathematical axioms and theorems.
+The proof assistant includes a GUI for creating proofs,
+proof verification tools, and grammatical/sytnax analysis.
+
+mmj2 was originally developed by Mel O'Cat 2005-2011.
+This version has been further modified by Mario Carneiro and David A. Wheeler.
+This program is released under the terms of the
+GNU General Public License Version 2 or any later version
+(SPDX license identifier GPL-2.0-or-later); see [LICENSE.TXT][].
+
+## Installation
+
+Refer to [INSTALL.html][] for instructions on how to install mmj2.
+This includes installing a Java runtime if you don't already have one
+(since mmj2 requires a Java runtime).
+
+## Interactive Tutorial
+
+Once you have installed mmj2, if you've not used mmj2 before,
+be sure to try the interactive (which ought to take about an hour).
+ [mmj2 Proof Assistant Tutorial][6]
+
+## Quick Start
+
+See [Quick Start]([QuickStart.html) if you want to quickly get running.
+
+## More information
+
+Once you've installed mmj2, the following pages may be useful:
+
+* [mmj2 Proof Assistant User Guide](doc/PAUserGuide/Start.html)
+* [General information about mmj2](mmj2.html)
+* [mmj2 Command Line Arguments](doc/mmj2CommandLineArguments.html)
+
+## Contributing
+
+If you want to contribute to this version of mmj2,
+please submit issues, pull requests, etc., to:
+<https://github.com/digama0/mmj2>
+
+MMj2 was initially developed by Mel O'Cat.
+If you wish to provide financial support to Mel O'Cat,
+mmj2's original developer, send donations via Paypal to
+*siskiyousis at gmail.com*.
+To send questions, bugs, enhancements, etc. about the origina mmj2,
+contact Mel at *X178G243 at yahoo.com*.
```

Comment:
Should we remove / comment out this section? Do these addresses even work?

Reply:
I don't know, but I hesitate to remove them.

Reply:
I think we should keep them. Mel did a lot of work, I would like him to retain credit.

Reply:
I think we should credit Mel of course, but the contact information sounds years out of date. I would be more comfortable with a list of people who have contributed to mmj2, without a request for donations.

Reply:
I moved the detailed information about Mel O'Cat to a separate file, with a link to it from the README. I am *NOT* comfortable doing anything more than that. I suspect it's allowed *legally*, but I would feel wrong removing it more than that. I suspect the email address is still valid, actually.

Reply:
So I think that issue (Mel O'CaT contact info) is completely resolved.